index.html

```diff
@@ -1,188 +1,202 @@
+                    <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="Mail icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\github icon.svg" alt="Github icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="Peace icon"></a></li>
                 </ul>
             </section>
         </section>
         <section id="work">
-            <!-- projecto 1 -->
+            <!-- project 1 -->
             <section class="project">
-                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="Project picture">
+                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="First project card picture">
                 <h2>Tonic</h1>
-                <section>
-                    <ul class="canopy">
-                        <li>CANOPY</li>
-                        <li><img src="img\Counter.svg"></li>
-                        <li><span class="lightgray">Back End Dev</span></li>
-                        <li><img src="img\Counter.svg"></li>
-                        <li><span class="lightgray">2015</span></li>
+                    <section>
+                        <ul class="canopy">
+                            <li>CANOPY</li>
+                            <li><img src="img\Counter.svg" alt="Dot icon"></li>
+                            <li><span class="lightgray">Back End Dev</span></li>
+                            <li><img src="img\Counter.svg" alt="Dot icon"></li>
+                            <li><span class="lightgray">2015</span></li>
+                        </ul>
+                    </section>
+                    <p>
+                        A daily selection of privately personalized
+                        reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programmingLanguages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
                     </ul>
-                </section>
-                <p>
-                    A daily selection of privately personalized 
-                    reads; no accounts or sign-ups required.
-                </p>
-                <ul class="programmingLanguages">
-                    <li>html</li>
-                    <li>css</li>
-                    <li>javaScript</li>
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
-            <!-- projecto 2 -->
+            <!-- project 2 -->
             <section class="project">
-                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 2.svg" alt="Project picture">
+                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 2.svg" alt="Second project card picture">
                 <h2>Multi-Post Stories</h1>
```

Comment:
- [ ] The page has at least one heading `<h1>`, and here you use two different `hx` in one tag please kindly fix that 🙃. `eg: <h2>Multi-Post Stories</h2>` 👍 👍